force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -44,7 +50,8 @@ public with sharing class VariablesDataTypesOperators {
      * @return The difference between the two numbers.
      */
     public static Integer subtraction(Integer a, Integer b) {
-        return null; // Replace null with the variable you used to store the result
+        Integer diff1 = a - b;
+        return diff1; // Replace null with the variable you used to store the result
```

Comment:
You could in-line this and get rid of a variable you don't need. 
return a - b;